src/bin/pg_autoctl/cli_get_set_properties.c

```diff
@@ -32,17 +35,21 @@ static bool set_formation_number_sync_standbys(Monitor *monitor,
 CommandLine get_node_replication_quorum =
 	make_command("replication-quorum",
 				 "get replication-quorum property from the monitor",
-				 CLI_PGDATA_USAGE,
-				 CLI_PGDATA_OPTION,
-				 cli_getopt_pgdata,
+				 " [ --pgdata ] [ --json ] [ --name ]",
```

Comment:
Seems like this one still needs `--formation`

Reply:
Fixed.